src/components/Table/Styled.jsx

```diff
@@ -92,6 +92,17 @@ export const StyledBody = styled.div`
       }
     }
   }` : '')};
+
+  ${props => (props.highlightUser !== undefined && props.highlightUser !== null ? `
```

Comment:
to make this more readable maybe the style should just be applied directly to the row?  nesting css selectors makes the code more difficult to understand.